frontend/src/components/Map/CrossingStaticMap.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import ReactMapboxGl, { Marker } from "react-mapbox-gl";
+import mapboxstyle from 'components/Map/mapboxstyle.json';
+import { statusIcons } from 'constants/StatusConstants';
+
+const Map = ReactMapboxGl({ accessToken: null, interactive: false });
+
+class CrossingStaticMap extends Component {
+
+  onStyleLoad = (map) => {
+    map.resize();
```

Comment:
Was this secret to centering the pin after it loads off-center?

Wondering if this may be requesting the map tiles twice. Seems like there are duplicate requests in the Network tab and I see some discussion around loading that map without a tile source and then adding the tile source with the onStyleLoad callback. https://github.com/alex3165/react-mapbox-gl/issues/22